src/Form/UnregistrationAdherentType.php

```diff
@@ -9,7 +9,7 @@
 use Symfony\Component\Form\FormInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class UnregistrationType extends AbstractType
+class UnregistrationAdherentType extends AbstractType
```

Comment:
idem, I think it can be the same class for two type of unregistration because it has juste the choices option was changed